orchagent/vxlanorch.cpp

```diff
@@ -675,6 +1326,35 @@ bool VxlanTunnelOrch::removeVxlanTunnelMap(string tunnelName, uint32_t vni)
         return false;
     }
 
+    // Update the map count and if this is the last mapping entry 
+    // make SAI calls to delete the tunnel and tunnel termination objects.
+
+    tunnel_obj->vlan_vrf_vni_count--;
```

Comment:
This may break the existing vxlan decap functionality as mappers are add/removed but the tunnel is never removed

Reply:
If the vlan_vrf_vni_count becomes non zero then the tunnel/terminator is created again. So didnt understand why the functionality will be broken. We would need atleast 1 mapping entry for decap functionality If I understand the feature correctly.